packages/angular/package.json

```diff
@@ -25,24 +25,34 @@
     "@sentry/types": "6.18.0",
     "@sentry/utils": "6.18.0",
     "rxjs": "^6.6.0",
-    "tslib": "^1.9.3"
+    "tslib": "^2.0.0"
```

Comment:
Does it make sense to add Rxjs 7 support as it is with Angular itself? Another question, do you still have node 6 as minimum? Angular drop support for it
https://github.com/angular/angular/blob/master/packages/core/package.json

Reply:
Hey, thanks for writing in! 
I agree with you that if we just had to support the most recent Angular version, we could bump Rxjs and Node but we want to support Angular 10 and upwards which afaik requires us to stick with Rxjs 6. For Node, Angular 10 doesn't limit Node support, at least not as far as I can see from `package.json` here: https://github.com/angular/angular/blob/10.0.x/packages/core/package.json

If you have a suggestion on how we could handle Angular version support better (v10 upwards), I'd be happy to read it though.

Reply:
You can look into Angular\Node matrix here (https://www.redcricket.net/en/news-en/compatibility-list-for-angular-angular-cli-and-node-js/) 
Support for Node 10 has beeen dropped last year https://github.com/angular/angular/pull/41544

Re:Rxjs, I just want to see Rxjs7 be allowed, as now Angular 13 allow using for newer version

Reply:
I looked into this and according to the matrix, Angular 10 supported Node 10. So I guess we could bump node to Node 10 for @sentry/angular but I'll check with my team first.

Regarding Rxjs, I think we can get away with allowing both, 6 and 7 as peer dependencies rather than rxjs 6 as a direct dependency (which is recommended by the Angular library creation guide anyway). Again, I'll check with the team first.